src/components/filters/Filters.jsx

```diff
@@ -0,0 +1,16 @@
+import { Container } from './styles';
+import CategoryFilter from 'components/categoryFilter/CategoryFilter';
+import SizeFilter from 'components/sizeFilter/SizeFilter';
+import PriceFilter from 'components/priceFilter/PriceFilter';
+
+const Filters = () => {
+  return (
+    <Container>
```

Comment:
```suggestion
    <FiltersContainer>
```

Reply:
Изменил `Container` на `FiltersContainer`
```suggestion
    <FiltersContainer>
```